src/Pipedrive.net/Models/Request/OAuthRefreshTokenRequest.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Pipedrive.Helpers;
+using Pipedrive.Internal;
+
+namespace Pipedrive
+{
+    public class OAuthRefreshTokenRequest : RequestParameters
+    {
+        public OAuthRefreshTokenRequest(string clientId, string clientSecret, string refreshToken)
+        {
+            Ensure.ArgumentNotNullOrEmptyString(clientId, nameof(clientId));
+            Ensure.ArgumentNotNullOrEmptyString(clientSecret, nameof(clientSecret));
+            Ensure.ArgumentNotNullOrEmptyString(refreshToken, nameof(refreshToken));
+
+            ClientId = clientId;
+            ClientSecret = clientSecret;
+            RefreshToken = refreshToken;
+        }
+
+        [Parameter(Key = "refresh_token")]
+        public string RefreshToken { get; }
+
+        [Parameter(Key = "grant_type")]
+        public string GrantType => "refresh_token";
```

Comment:
@DavidRouyer 
What do you think of this approach?
The grant type could not be another but refresh_token in this scenario.